Week3/homework/exercise2.js

```diff
@@ -0,0 +1,11 @@
+"use strict"
+
+let dogage = []
+function calculateDogAge(num){
+    let realdogage = num * 7;
```

Comment:
`const` instead of `let` and `realDogAge` instead of `realdogage` please. Try to read this article again to revisit topic of variable naming conventions - https://github.com/HackYourFuture/fundamentals/blob/master/fundamentals/naming_conventions.md